packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -280,38 +280,38 @@ In the Okta event data object, the UserAgent object provides specifications abou
 | geolocation | Contains the geolocation coordinates (latitude, longitude)                                                           | [Geolocation object](#geolocation-object) | TRUE     |
-| city        | The city encompassing the area containing the geolocation coordinates, if available (e.g. Seattle, San Francisco)    | String                                    | TRUE     |
-| state       | Full name of the state/province encompassing the area containing the geolocation coordinates (e.g. Montana, Incheon) | String                                    | TRUE     |
-| country     | Full name of the country encompassing the area containing the geolocation coordinates (e.g. France, Uganda)          | String                                    | TRUE     |
-| postalCode  | Postal code of the area encompassing the geolocation coordinates                                                     | String                                    | TRUE     |
+| city        | The city that encompasses the area that contains the geolocation coordinates, if available (for example, Seattle, San Francisco)    | String                                    | TRUE     |
+| state       | Full name of the state/province that encompasses the area that contains the geolocation coordinates (for example, Montana, Incheon) | String                                    | TRUE     |
+| country     | Full name of the country that encompasses the area that contains the geolocation coordinates (for example, France, Uganda)          | String                                    | TRUE     |
+| postalCode  | Postal code of the area that encompasses the geolocation coordinates                                                     | String                                    | TRUE     |
 
 ### Geolocation object
 
-Latitude and longitude of the geolocation where an action was performed, formatted according to the [ISO-6709](https://en.wikipedia.org/wiki/ISO_6709) standard.
+The latitude and longitude of the geolocation where an action was performed that is formatted according to the [ISO-6709](https://en.wikipedia.org/wiki/ISO_6709) standard
 
 | Property   | Description                                                                                       | DataType        | Nullable |
 | ---------- | ------------------------------------------------------------------------------------------------- | --------------- | -------- |
-| lat        | Latitude. Uses 2 digits for the [integer part](https://en.wikipedia.org/wiki/ISO_6709#Latitude)   | Double          | FALSE    |
-| lon        | Longitude. Uses 3 digits for the [integer part](https://en.wikipedia.org/wiki/ISO_6709#Longitude) | Double          | FALSE    |
+| lat        | Latitude. Uses two digits for the [integer part](https://en.wikipedia.org/wiki/ISO_6709#Latitude).   | Double          | FALSE    |
```

Comment:
No periods in table descriptions.

Reply:
Since there was another fragment before this fragment, I left the periods in there. Hope that's OK. That seems to be what we're doing in other pages I've noticed.

Reply:
@christinechoy-okta - We use the period when the fragment is followed by a complete sentence. That's not what is going on here. I would change this to the following:

Latitude: Uses two digits for the blah blah blah with no period.